Gruntfile.js

```diff
@@ -36,25 +36,51 @@ module.exports = function (grunt) {
       },
       concat: {
         options: {
-          separator: ';'
+          separator: '\n'
         },
         ui: {
-          src: ['src/**/*UI.module.js', 'dist/scripts/templates.js', 'src/components/**/*.js', 'src/filters/**/*.js', 'src/ui-services/**/*.js'],
+          src: [
+            'src/pluginLoader.js',
```

Comment:
Pondering whether `pluginLoader` fits better under `services` over `ui`, or if it should be in both.  We consume the `dist`, so I don't think it can break the console, loading it twice *probably* overwrites one version, but its hard to tell if this thing has any internal `caches` that would cause memory leaks.

Reply:
From what I could tell, these files are intended to be consumed individually and not together. If you only need UI components, you don't have to consume the Services file and vice versa, and if you need both you just consume the origin-web-common.js file.

Reply:
The other option would be to just leave it out of the concatenated files, then leave it to the consumer to load the pluginLoader script before the others.